build.sbt

```diff
@@ -640,7 +640,8 @@ lazy val publishableSettings = Def.settings(
     platformDepsCrossVersion.value == ScalaNativeCrossVersion.binary,
   publishTo := Some {
     if (isCustomRepository) "adhoc" at adhocRepoUri
-    else if (isSnapshot.value) Opts.resolver.sonatypeSnapshots
+    // NOTE: isSnapshot.value does not work with sbt-dynver
+    else if (version.value.endsWith("SNAPSHOT")) Opts.resolver.sonatypeSnapshots
```

Comment:
Looks like the same change will have to be made to the Scalafix build.

Reply:
This is only necessary when publishing to maven from local machine. Scalafix is only released via ci.